.travis.yml

```diff
@@ -15,7 +15,7 @@ services:
 language: go
 branches:
   only:
-    - master
+    - release-v3.0
```

Comment:
should this not be `release-v3.1` in addition to `master`?

Reply:
yeah.. addressed @nixpanic